### ISSUE TITLE
Storybook doc page scrolling down

### ISSUE THREAD

**Issue by andresin87** [adevinta/spark#1849]
### Description

When opening dropdown storybook doc page it scrolls down to the first story

### Steps to reproduce

1. Go to ['dropdown storybook doc page'](https://sparkui.vercel.app/?path=/docs/experimental-dropdown--docs)
4. It scrolls down to the first story automatically

### Spark Component and Version

@spark-ui/dropdown

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by Powerplex** [comment]
@andresz1 I investigated this, and in fact it's related to Downshift. It is caused by the "ControlledOpenState" Story where the dropdown is opened by default.

Inside `DropdownTrigger`, the ref passed if a combination of the forwardedRef and the downshift ref for the toggle button.

```
const ref = useMergeRefs(forwardedRef, downshiftRef)
```

If I replaced with only the forwarded ref the `autoScroll` behaviour is off:
```
const ref = useMergeRefs(forwardedRef, downshiftRef)
```

For now I can't find a way to disable it, normally the `autoFocus={false}` should do the trick, but it's not.
